ocp_install_env.sh

```diff
@@ -97,6 +97,15 @@ EOF
     fi
 }
 
+function additional_trust_bundle() {
+    if [ ! -z "$ADDITIONAL_TRUST_BUNDLE" ]; then
+cat <<EOF
+additionalTrustBundle: |
+  $(echo ${ADDITIONAL_TRUST_BUNDLE} | sed -e ':a;N;$!ba;s/\n/\n  /g')
```

Comment:
This will create a duplicate section when `image_mirror_config` is called (when MIRROR_IMAGES or ipv6 is specified), we'll need some way to merge the content.

Reply:
Why duplicated?does `image_mirror_config` adds this section?

Reply:
Yes exactly, it already adds this section https://github.com/openshift-metal3/dev-scripts/blob/master/utils.sh#L331